packages/storage-driver-cloudinary/src/index.ts

```diff
@@ -119,10 +123,12 @@ export class DriverCloudinary implements Driver {
 		return `Basic ${base64}`;
 	}
 
-	async read(filepath: string, range?: Range) {
+	async read(filepath: string, options?: ReadOptions) {
+		const { range, bustCache } = options ?? {};
+
 		const resourceType = this.getResourceType(filepath);
 		const fullPath = this.fullPath(filepath);
-		const signature = this.getParameterSignature(fullPath);
+		const signature = bustCache === true ? this.getLatestVersion() : this.getParameterSignature(fullPath);
```

Comment:
How often do we read without busting the cache here? While less efficient, it might be an option to always cache bust in cloudinary as it's unreliable otherwise, and leave the other drivers as is